lua/lazyvim/plugins/editor.lua

```diff
@@ -135,13 +135,16 @@ return {
       {
         "<leader>sr",
         function()
-          require("grug-far").grug_far({
-            prefills = {
-              search = vim.fn.expand("<cword>"),
-              filesFilter = "*." .. vim.fn.expand("%:e"),
-            },
+          local is_visual = vim.fn.mode():lower():find("v")
+          if is_visual then -- needed to make visual selection work
+            vim.cmd([[normal! v]])
+          end
+          local grug = require("grug-far");
```

Comment:
I've never seen this `;` at the end. What does it actually do? Chain statements so that they remain in same scope? Because when I removed it to see if it was a typo, lua_ls complained about `undefined global grug`.

Reply:
If you would remove that, then lua thinks the next line is a parameter for the function.
Would break lua syntax